docs/UserGuide.md

```diff
@@ -499,8 +508,10 @@ This command will help you keep track of which debts have been paid and which de
 #### Marking debts as unpaid: `unmark`
 
 **What it does:**<br>
-Marks the debts specified from a person in PayMeLah as unpaid. Specifying multiple debts will mark all those debts as unpaid. <br>
-This command will help you undo any mistake you made when marking a debt as paid.
+This command marks the debts specified from a person in PayMeLah as unpaid. Specifying multiple debts will mark all those debts as unpaid.
+
+**When to use:**<br>
+When you realise that a previously marked-as-paid debt has in fact not been repaid yet, you can use this command to indicate that it is still unpaid.
```

Comment:
When you realise that a debt you had previously [marked as paid](#...) was actually not paid yet, you can...